post-scan-actions/gcp-python-teams-notification/main.py

```diff
@@ -0,0 +1,117 @@
+import os
+import json
+import base64
+import urllib3
+from http.client import responses
+
+http = urllib3.PoolManager()
+
+def get_gcp_project_id(resource_name):
+    return str(resource_name).split("/")[1]
+
+def get_bucket_name_from_file_url(file_url):
+    return str(file_url.split("/")[-2:][0])
+
+def get_file_name_from_file_url(file_url):
+    return str(file_url.split("/")[-1:][0])
+
+def build_file_metadata_url(project_id, bucket_name, file_name):
+    return "https://console.cloud.google.com/storage/browser/_details/" + bucket_name + "/" + file_name + "?project=" + project_id
+
+def main(event, context):
+
+    try:
+
+        print(f'Context: {context}')
+        print(f'Event: {event}')
+        base64_data = event.get('data', '')
+        base64_bytes = base64_data.encode('ascii')
+        message_bytes = base64.b64decode(base64_bytes)
+        message = json.loads(message_bytes.decode('ascii'))
+        print(f'Message: {message}')
+
+        url = os.environ['TEAMS_URL']
+
+        print("""This Function was triggered by messageId {} published at {} to {}""".format(context.event_id, context.timestamp, context.resource))
```

Comment:
What are these `print` used for? If they are used for logging, I suggest use old style formatting according to [logging module doc](https://docs.python.org/3/howto/logging.html#optimization) ([Django doctor](https://blog.devgenius.io/when-python-old-style-string-formatting-is-best-practice-691a0759b37e) also provides a thorough explanation).
While they are used for merely printing messages, I suggest all `print` use f-string instead to make your code have a unified style.

Reply:
@luhsuanchen Appreciate your feedback. I intended to change everything to use the `logging` package but got lost in other FSS plugin dev. I will change it to `logging` with a commit to the branch right away.